assets/src/edit-story/components/header/buttons/publish.js

```diff
@@ -104,10 +111,10 @@ function Publish() {
         disabled={!capabilities?.hasPublishAction || isSaving || isUploading}
         text={text}
       />
-      <TitleMissingDialog
+      <ReviewChecklistDialog
         open={Boolean(showDialog)}
```

Comment:
This wasn't you - but I think `showDialog` is already a boolean, right?
```suggestion
        open={showDialog}
```

Reply:
it is! i'll update it!